CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ You can find and compare releases at the [GitHub release page](https://github.co
 
 ## Unreleased
 
+- Split up `@can` directive into `@canFind`, `@canModel`, `@canQuery`, `@canResolved` and `@canRoot` https://github.com/nuwave/lighthouse/pull/2483
+- Added `action` and `return_value` arguments to `@can*` family of directives https://github.com/nuwave/lighthouse/pull/2483
```

Comment:
We use `camelCase` for directive arguments.

Reply:
fixed it everywhere